lib/runners/processor/clang_tidy.rb

```diff
@@ -0,0 +1,106 @@
+module Runners
+  class Processor::ClangTidy < Processor
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+          apt: enum?(string, array(string)),
+          'compilation-options': object?(
+            '-I': enum?(string, array(string))
+          )
+        })
+      }
+
+      let :issue, object(
+        severity: string,
+      )
+    end
+
+    register_config_schema(name: :clang_tidy, schema: Schema.runner_config)
+
+    VALID_EXTENSIONS = [".c", ".cc", ".cpp", ".c++", ".cp", ".cxx"].freeze
+    GLOB_HEADERS = "**/*.{h,hpp,h++}".freeze
+
+    def analyze(changes)
+      deploy_packages
```

Comment:
[suggestion] It is better to use `setup`:

```suggestion
    def setup
      deploy_packages
      yield
    end

    def analyze(changes)
```

Reply:
Good catch! The `setup` method suits this purpose.
commit b3ac6bad627c92d2bd7a65fcc067a02025ac0898